pkg/minishift/addon/addon_meta.go

```diff
@@ -36,6 +38,7 @@ type AddOnMeta interface {
 	Description() []string
 	RequiredVars() []string
 	GetValue(key string) string
+	OpenShiftVersion() string
```

Comment:
Instead of a string why don't you create a struct for it? One field containing the version and on a range. This could make things a bit easier down the road. Also, you could detect the wrong usage of the version and range when the add-on is parsed, compared to when it is applied.